### ISSUE TITLE
Algo/largest_sum_contiguous_subarray [C++]

### ISSUE THREAD

**Issue by manish1997** [iiitv/algos#478]
#### This issue is about [Algo/DS Name](link to resource for the Algo/DS).

- [x] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
	- [ ] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by polmauri** [comment]
Can I claim this?

**Comment by mohitkyadav** [comment]
@manish1997 are you working on this?

**Comment by manish1997** [comment]
@mohitkyadav no. My coding style is different than what your rules want in c++.

**Comment by AbhishekKatara** [comment]
Thanks @yashLadha
